src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -185,7 +191,10 @@ export function DownloadOptionPanel (props) {
     };
 
     const dlTitle = get(dlParams, 'TitlePrefix', 'Download') + '-' + dlTitleIdx;
-
+    // IRSA-3944: SOFIA required a message to guide the user before downloading AOR such as:
+    // 'All levels associated with this AOR will be downloaded. To download individual data products, go to the relevant instrument tab.'
+    // PreTitleMessage can be added to the dlParams from the parent and passed here (FIREFLY-723)
```

Comment:
This message should probably be at the top with the other documentation

Reply:
good, i'll update it.